docs/running/wdl.rst

```diff
@@ -0,0 +1,139 @@
+WDL Support in Toil
+********
```

Comment:
Build errors for `make docs` complain about `WARNING: Title underline too short.`

This happens lots of times for all of the titles.

Reply:
Fixed.